core/src/main/java/org/primefaces/extensions/model/mongo/MorphiaLazyDataModel.java

```diff
@@ -249,6 +252,59 @@ public int count(final Map<String, FilterMeta> map) {
 
                                 break;
 
+                            case BETWEEN:
+                                if (metadata.getFilterValue() instanceof List) {
+                                    final List<?> dates = (List) metadata.getFilterValue();
+                                    if (dates.size() > 1) { // does this ever have less than 2 items?
+                                        q.filter(Filters.gte(field, dates.get(0)), Filters.lte(field, dates.get(1)));
+                                    }
+
+                                }
+                                break;
+                            case NOT_CONTAINS:
+                                q.filter(Filters.regex(field).pattern(val + Constants.EMPTY_STRING).caseInsensitive().not());
+
+                                break;
```

Comment:
Technically this passed Spotless, but can you please align blank lines with the other filters?